packages/helpers/classes/attachment.js

```diff
@@ -40,38 +40,65 @@ class Attachment {
     data = toCamelCase(data);
 
     //Extract properties from data
-    const {content, filename, type, disposition, contentId, file} = data;
-
-    let fileContent = !file ? content : this.readFile(file);
+    const {
+      content,
+      filename,
+      type,
+      disposition,
+      contentId,
+      filePath
+    } = data;
+
+    if ((typeof content !== 'undefined') && (typeof filePath !== 'undefined')) {
+      throw new Error(
+        'The props \'content\' and \'filePath\' cannot be used together.'
```

Comment:
Could you use template strings to make it a bit cleaner without the backslashes? E.g.

```
`The props 'content' and 'filePath' cannot be used together.`
```